types/plur/plur-tests.ts

```diff
@@ -0,0 +1,5 @@
+import plur from 'plur';
```

Comment:
The correct way to import this, based on the implementation, would be:

`import plur = require('plur')`

An explanation can be found in the readme: https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

> For an NPM package, `export =` is accurate if `node -p 'require("foo")'` is the export, and `export default` is accurate if `node -p 'require("foo").default'` is the export.